lib/action_controller/serialization.rb

```diff
@@ -6,7 +6,7 @@ module Serialization
 
     include ActionController::Renderers
 
-    ADAPTER_OPTION_KEYS = [:include, :fields, :root, :adapter]
+    ADAPTER_OPTION_KEYS = [:include, :fields, :adapter]
```

Comment:
I'll have to read through the code some more to make sure I understand the changes as they affect tests that were deleted or whose expectations were changed.  It didn't break https://github.com/rails-api/active_model_serializers/pull/954 (once I rebased it), but I think this is the kind of thing we should include in the README between 'how it works' and 'intended usage'...